packages/cli/src/lib/colors.ts

```diff
@@ -3,4 +3,6 @@ import ansiColors from 'ansi-colors';
 export const colors = {
   dim: ansiColors.dim,
   bold: ansiColors.bold,
+  green: ansiColors.green,
+  whiteBright: ansiColors.whiteBright,
```

Comment:
I think this green is the same as `outputToken.green`, right? I see you've used both but perhaps to stick to `outputToken.green` instead of `colors.green`.

(This file is just a workaround to cli-kit colors and should be used sparingly so that we keep consistent style across CLIs).

Reply:
I'll have a look! I was getting very frustrated at the lack of exports from `cli-kit`.

Reply:
Ah, not sure 🤔 

@matteodepalo Can we suggest adding some sort of `dim` to `outputToken`? It seems to be quite useful for showing secondary information.

Reply:
The last news I got was that it would be nice to keep the same type of colors for CLIs so we were trying to use `outputToken`. However, we can start migrating to `colors` direclty 👍 -- but note that this `colors` is a local `ansi-colors`, not anything exported from CLI-kit 🤔